src/Identifier/Ldap/ExtensionAdapter.php

```diff
@@ -93,7 +93,11 @@ public function getConnection()
     public function connect($host, $port, $options)
     {
         $this->_setErrorHandler();
-        $this->_connection = ldap_connect($host, $port);
+        $resource = ldap_connect($host, $port);
+        if ($resource === false) {
+            throw new RuntimeException('Unable to connect to LDAP server.');
+        }
+        $this->_connection = $resource;
         $this->_unsetErrorHandler();
 
         if (is_array($options)) {
```

Comment:
Do we need this check? Shouldnt we typehint or throw meaningful exception instead of ignoring non array $options ? :)

Reply:
Adding / fixing type hints in beyond the scope of this PR.